files/build_templates/sonic_debian_extension.j2

```diff
@@ -89,6 +89,16 @@ sudo cp -f $IMAGE_CONFIGS/bash/bash.bashrc $FILESYSTEM_ROOT/etc/
 sudo dpkg --root=$FILESYSTEM_ROOT -i target/debs/sonic-device-data_*.deb || \
     sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
 
+# Install pam-tacplus
```

Comment:
Is it possible to capsulate the TACACS+ functionality into a docker container like docker-vas? It is not an easy a job but technically feasible.